### PR TITLE
Add loading spinner and display it when results are loading

### PR DESCRIPTION
**Pull request by mok4ry**
This gets rid of the "No Results to Display" problem in Issue #81.

However, the sizing of the page is a little awkward with the results being replaced by a spinner and the main content div height changing so drastically. I'd rather address that in a separate issue/PR.

### CODE REVIEW

**Comment by lukecoy** [comment]
can we get a pic or is that kind of hard to do?

**Comment by mok4ry** [comment]
Yeah, I can put up a pic. Also I go back on my "I'd rather address [the awkward height change] issue in a separate PR" opinion and will attach some styling adjustments to this PR tomorrow.

**Comment by mok4ry** [comment]
loading-spinner
cc @amb8805 @lukecoy @Lettuceman44

EDIT: gif is jumpy and grainy, real animation is very smooth.

**Comment by lukecoy** [comment]
Saw it in person, can confirm

File: public/javascripts/containers/Results.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { connect } from 'react-redux';
 import Result from '../components/Result';
 import Pagination from './Pagination';
+import LoadingSpinner from '../components/LoadingSpinner';
```
Comment (lukecoy) [inline public/javascripts/containers/Results.jsx:5]: the style guide doesn't complain about the file names being different case?

File: public/stylesheets/components/LoadingSpinner.css [hunk 2]
```diff
@@ -0,0 +1,292 @@
+/*!
+ * Load Awesome v1.1.0 (http://github.danielcardoso.net/load-awesome/)
+ * Copyright 2015 Daniel Cardoso <@DanielCardoso>
+ * Licensed under MIT
+ */
```
Comment (lukecoy) [inline public/stylesheets/components/LoadingSpinner.css:5]: oh nice!

**Comment by lukecoy** [comment]
looks good, I wonder about the file name case but yea

**Comment by mok4ry** [comment]
What do you mean different case filenames? Like how some of them are `lowerUpper` and some are `UpperUpper`?

**Pull request merged**